src/main/default/lwc/lookup/lookup.js

```diff
@@ -112,15 +119,16 @@ export default class Lookup extends LightningElement {
 
         // Compare clean new search term with current one and abort if identical
         const newCleanSearchTerm = newSearchTerm.trim().replace(/\*/g, '').toLowerCase();
-        if (this._cleanSearchTerm === newCleanSearchTerm) {
+
+        if (!this.allowEmptySearchQuery && this._cleanSearchTerm === newCleanSearchTerm) {
```

Comment:
we aren't working with empty search terms anymore I thought?

Reply:
by default, we are not but if someone wants to work with an empty string, we allow it by using the `allowEmptySearchQuery` api property